cmd/jaeger/internal/command_test.go

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2024 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package internal
+
+import (
+	"testing"
+
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCommand(t *testing.T) {
+	cmd := Command()
+
+	assert.NotNil(t, cmd)
+	assert.Equal(t, "jaeger", cmd.Use)
+	assert.Equal(t, description, cmd.Long)
+
+	runE := cmd.RunE
+	assert.NotNil(t, runE)
+}
+
+func TestCheckConfigAndRun_DefaultConfig(t *testing.T) {
+	cmd := &cobra.Command{
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return nil
+		},
+	}
+	cmd.Flags().String("config", "", "path to config file")
+	getCfg := func(name string) ([]byte, error) {
+		return []byte("default-config"), nil
+	}
+	runE := func(cmd *cobra.Command, args []string) error {
+		return nil
+	}
+
+	err := checkConfigAndRun(cmd, nil, getCfg, runE)
+	require.NoError(t, err)
+
+	configFlag := cmd.Flag("config")
+	assert.Equal(t, "yaml:default-config", configFlag.Value.String())
+	assert.False(t, configFlag.Changed)
```

Comment:
what are you testing with this?

Reply:
i was trying to test if the config flag is changes, but removed them now